.gitignore

```diff
@@ -2,3 +2,6 @@ certs/*
 spec/spec
 examples/simple-consumer/simple-consumer
 examples/simple-producer/simple-producer
+
+# GoLand IDE project files
+/.idea
```

Comment:
this should be in your global .gitignore

Reply:
I made the ignore to be a bit more fine-grained and follow JetBrains' guidelines. It makes sense to me to ignore project-specific files in a project-specific `.gitignore`.